show/vnet.py

```diff
@@ -333,6 +333,29 @@ def routes():
     """Show vnet routes related information"""
     pass
 
+def pretty_print(table, r, epval, mac_addr, vni, state):
```

Comment:
@dgsudharsan , do you know if there are any existing utilities that we can reuse for this?